### DISCUSSION TITLE
Any way to change number outputs for auto numbering?

### DISCUSSION THREAD

**Discussion by haptiqblack** [yermak/AudioBookConverter#208] [Q&A]
I notice it always forces the numbers to be XXX so I get "001" and "010" is there any way to change this so I can just get the single number with my chapters? So I can have "Chapter 1" and "Chapter 10" instead of having the extra zeros?

**Comment by yermak** [comment]
I will have a look at this.

Currently it's not possible, but I may be able to make it configurable

**Comment by yermak** [comment]
@haptiqblack ,
starting from version 5.5.14 you should be able to do so, find AudioBookConverter.property file, normally locatated in ApplicationData folder.
In section chapter_format find `<CHAPTER_NUMBER; format\="%,03d"> ` - 03 means how total numbers it will have, change to 01d and restart the program.
Same could be done with chapter and book output name.

**Reply by haptiqblack** [reply]
Awesome that’s great! Thank you!

**Reply by albert239825** [reply]
Hello, I can't find the file. Is this the correct path? C:\Users\[user]\AppData\Local\AudioBookConverter

Thanks!

**Reply by PepperRobb97** [reply]
Albert
I've had the same issue. Unfortunately, this is an issue for some of us. If you are on the current version and you can't find the properties file, you may have to revert to a previous version. The only version I was able to successfully get this to work on (and everything works great) is [version 5.6.2](https://github.com/yermak/AudioBookConverter/releases/tag/version_5.6.2) portable. Extract the portable file anywhere you want. The properties file SHOULD appear in the following folder :

C:\Users\username\AppData\Roaming\AudioBookConverter-5.6.2

I've actually had some issues where the properties file doesn't populate even after running the program the first time. But, I've attempted to paste a backup copy of my properties file where it should go and it seems to work. Here is a copy of properties file. I have already adjusted some of the settings in the properties to fix some common issues that I've had (fixed the space in the title, 2 zeros instead of 3 in the chapter, series name in the filename). 
I hope this helps you out!


[AudioBookConverter-5.6.2.zip](https://github.com/yermak/AudioBookConverter/files/12060284/AudioBookConverter-5.6.2.zip)

**Reply by albert239825** [reply]
Thanks @PepperRobb97 for the info. Honestly, that might be too much of a hassle for me. It would be really nice if there was a place within the app where we could specify the format.
